### ISSUE TITLE
Internal error trying to create account

### ISSUE THREAD

**Issue by kevinherron** [faircodeio/faircode-license#6]
When creating an account I get a page with nothing but "Internal error" after clicking the signup button.

**Comment by FredrikNoren** [comment]
@kevinherron Hm yeah it's a bug; I think it's because you probably have a stripe account associated with that email already? I'll see if I can make a fix for it (but will take some days as I'm traveling). In the meantime easiest "fix" is to register with a different email (if this is indeed the bug).

**Comment by kevinherron** [comment]
Yes, I do have a Stripe account associated with the email I used.
